quartz/components/Explorer.tsx

```diff
@@ -112,12 +112,12 @@ export default ((userOpts?: Partial<Options>) => {
             <polyline points="6 9 12 15 18 9"></polyline>
           </svg>
         </button>
-        <div id="explorer-content">
+        <nav id="explorer-content">
```

Comment:
eh pretty nit, but I'm indifferent about this

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu

Reply:
make ul as menu element in explorer